features/networking/ovn_ipsec.feature

```diff
@@ -157,3 +157,68 @@ Feature: OVNKubernetes IPsec related networking scenarios
     # Example ESP packet un-encrypted will look like 16:37:16.309297 IP ip-10-0-x-x.us-east-2.compute.internal > ip-10-0-x-x.us-east-2.compute.internal: ESP(spi=0xf50c771c,seq=0xfaad)
     And the output should match: 
        | <%= cb.workers[0].name %>.* > <%= cb.workers[1].name %>.*: ESP |
+       
+  # @author weliang@redhat.com
+  # @case_id OCP-40569
+  @admin
+  Scenario: Allow enablement/disablement ipsec at runtime
+    Given the env is using "OVNKubernetes" networkType
+    Given I store all worker nodes to the :workers clipboard
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    Given I have a project
+    Given I obtain test data file "networking/pod-for-ping.json"
+    When I run oc create over "pod-for-ping.json" replacing paths:
+      | ["spec"]["nodeName"] | <%= cb.workers[1].name %> |
+      | ["metadata"]["name"] | pod-worker1               |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+       | name=hello-pod |
+    And evaluation of `pod.ip_url` is stored in the :test_pod_worker1 clipboard
+    
+    Given I obtain test data file "networking/pod-for-ping.json"
+    When I run oc create over "pod-for-ping.json" replacing paths:
+      | ["spec"]["nodeName"]                 | <%= cb.workers[0].name %>                                                                     |
+      | ["metadata"]["name"]                 | pod-worker0                                                                                   |
+      | ["spec"]["containers"][0]["command"] | ["bash", "-c", "for f in {0..3600}; do curl <%= cb.test_pod_worker1 %>:8080 ; sleep 1; done"] |
```

Comment:
not sure if we want a curl `--connect-timeout  5` here, in case there is a pod-for-ping issue but IPsec is still working?  Just to guarantee a steady traffic flow even if there is some error?

Reply:
Beside the new steps of enable/disable ipsec, all other steps are copied from https://github.com/openshift/verification-tests/blob/master/features/networking/ovn_ipsec.feature#L115, wait the response from @anuragthehatter

Reply:
+1.    better to add the timeout

Reply:
Fixed to add --connect-timeout 5